contrib/natss/pkg/dispatcher/dispatcher/dispatcher.go

```diff
@@ -83,10 +107,60 @@ func createReceiverFunction(s *SubscriptionsSupervisor, logger *zap.SugaredLogge
 }
 
 func (s *SubscriptionsSupervisor) Start(stopCh <-chan struct{}) error {
+	// Starting Connect to establish connection with NATS
+	go s.Connect(stopCh)
+	// Trigger Connect to establish connection with NATS
+	s.signalReconnect()
 	s.receiver.Start(stopCh)
 	return nil
 }
 
+func (s *SubscriptionsSupervisor) connectWithRetry(stopCh <-chan struct{}) {
+	// re-attempting evey 60 seconds until the connection is established.
+	ticker := time.NewTicker(60 * time.Second)
```

Comment:
Please  change it to 1s, 60s is too big and it's the only place which creates a new connection.

Reply:
done, but do you think 1 second is not too aggressive?